sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/tests/ScenarioTests/SqlResourcesOperationsTests.cs

```diff
@@ -246,6 +255,120 @@ public void SqlCRUDTests()
             }
         }
 
+        [Fact]
+        public void SqlRoleTests()
+        {
+            var handler1 = new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK };
+            using (MockContext context = MockContext.Start(this.GetType()))
+            {
+                // Create client
+                CosmosDBManagementClient cosmosDBManagementClient = CosmosDBTestUtilities.GetCosmosDBClient(context, handler1);
+
+                SqlRoleDefinitionCreateUpdateParameters sqlRoleDefinitionCreateUpdateParameters = new SqlRoleDefinitionCreateUpdateParameters
+                {
+                    RoleName = "roleName",
+                    Type = RoleDefinitionType.CustomRole,
+                    AssignableScopes = new List<string>
+                    {
+                        string.Format("/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.DocumentDB/databaseAccounts/{2}", cosmosDBManagementClient.SubscriptionId, resourceGroupName, databaseAccountName)
```

Comment:
This assignable scope should have databaseAccountName2

Reply:
Added